kubernetes/loculus/templates/loculus-database-standin.yaml

```
@@ -5,7 +5,7 @@ kind: Deployment
 metadata:
   name: loculus-database
   annotations:
-    argocd.argoproj.io/sync-options: Replace=true
+    argocd.argoproj.io/sync-options: Replace=false
```

To revert before merging:
```suggestion
    argocd.argoproj.io/sync-options: Replace=true
```